src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19668.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue19668 : _IssuesUITest
+{
+	public Issue19668(TestDevice testDevice) : base(testDevice)
+	{
+	}
+	public override string Issue => "BoxView Placement inside Border.";
+
+	[Test]
+	[Category(UITestCategories.Border)]
+	public void BoxViewPlacementInsideBorder()
+	{
+		App.WaitForElement("Label");
+		VerifyScreenshot();
```

Comment:
Pending snapshot on some platforms. Already available in the latest build.

Example, Mac:
![image](https://github.com/user-attachments/assets/8d20282f-61bd-45ad-bb2e-5fca3a1bf44b)

Could you commit the images?